src/SectionTwo.tsx

```diff
@@ -0,0 +1,28 @@
+import Card from "./components/Card";
+import exclamationmark from "./assets/exclamationmarkk.png";
+import thumbsDown from "./assets/thumbs-down.png"
+import clock from "./assets/clock.png"
+import questionMark from "./assets/questionmark.png"
+
+
+export const SectionTwo = () => {
+  return (
+    <div className="container  px-5 md:px-10 lg:15px xl:px-20 md:py-20 xl:py-30 mx-auto">
+      <div className="">
+        <h1 className=" mb-5 text-2xl  md:text-4xl xl:text-[56px] text-center lg:text-left font-semibold tracking-tight text-[#1B1F3B]">Physical access is still stuck in the past </h1>
+        <p className="mb-7 text-sm text-center lg:text-left md:text-sm xl:text-lg/[28px] text-[#333858]">...and it's a cumbersome and draining as ever</p>
+        <div className="flex gap-5 flex-col lg:flex-row">
+      <div className="flex flex-col gap-5 mt-10">
+      <Card  header="Lost Keys & Cards"  content="Keys and Access Cards get misplaced often, -and they are expensive to replace." imageSrc ={exclamationmark} bgColor="bg-gradient-to-r from-[#88C4C8] via-[#E3FCFC] to-[#F6FFFD]" headerColor="text-[#1B1F3B]" color="text-[#333858]"/>
```

Comment:
replace the bg color for this card gradient to this

`bgColor="bg-gradient-to-r from-[#BFD6FA] to-[#E8F0FC]"`